CHANGELOG.md

```diff
@@ -324,33 +504,33 @@ Same changes as in `@mui/x-data-grid-pro@7.0.0-beta.4`.
 - The headless field hooks (e.g.: `useDateField`) now returns a new prop called `enableAccessibleFieldDOMStructure`.
   This property is utilized to determine whether the anticipated UI is constructed using an accessible DOM structure. Learn more about this new [accessible DOM structure](/x/react-date-pickers/fields/#accessible-dom-structure).
 
- When building a custom UI, you are most-likely only supporting one DOM structure, so you can remove `enableAccessibleFieldDOMStructure` before it is passed to the DOM:
+  When building a custom UI, you are most-likely only supporting one DOM structure, so you can remove `enableAccessibleFieldDOMStructure` before it is passed to the DOM:
 
-  ```diff
-    function MyCustomTextField(props) {
-      const {
-  +     // Should be ignored
-  +     enableAccessibleFieldDOMStructure,
-        // ... rest of the props you are using
-      } = props;
-
-      return ( /* Some UI to edit the date */ )
-    }
-
-    function MyCustomField(props) {
-      const fieldResponse = useDateField<Dayjs, false, typeof textFieldProps>({
-        ...props,
-  +     // If you only support one DOM structure, we advise you to hardcode it here to avoid unwanted switches in your application
-  +     enableAccessibleFieldDOMStructure: false,
-      });
-
-      return <MyCustomTextField ref={ref} {...fieldResponse} />;
-    }
-
-    function App() {
-      return <DatePicker slots={{ field: MyCustomField }} />;
-    }
-  ```
+```diff
```

Comment:
If I'm not mistaken, this section should have remained with the original indentation as well, right @flaviendelangle ? 🤔